Client/mods/deathmatch/logic/CClientSound.h

```diff
@@ -39,6 +39,9 @@ class CClientSound : public CClientEntity
     void SetPaused(bool bPaused);
     bool IsPaused(void);
 
+    void SetLooped(bool bLoop);
+    bool IsLooped(void);
```

Comment:
Replace `(void)` with `()` and make the member function **const**: `bool IsLooped() const;`